bin/heat-flow-module-L0/8_cooling_of_oceanic_lithosphere.ipynb

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
<p>Related to error in the cell above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>